common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -3099,6 +3099,12 @@
     ],
     "sqlState" : "42803"
   },
+  "MISSING_WINDOW_SPECIFICATION" : {
+    "message" : [
+      "Window specification <windowName> is not defined in the WINDOW clause."
```

Comment:
shall we make the error message more actionable? like "Please make sure it is defined in the WINDOW clause or verify its spelling."